pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -435,7 +437,7 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional withUkify (python3Packages.python.withPackages (ps: with ps; [ pefile ]))
   ;
 
-  #dontAddPrefix = true;
+  mesonBuildType = "release";
```

Comment:
Isn't this the default?

Reply:
The default is 'plain'. I wonder whether we should change it.